docs/getting-started/istio/index.md

```diff
@@ -149,3 +149,8 @@ spec:
 
 As the Rollout progresses through steps, the HTTP route destination weights will be adjusted to
 match the current setWeight of the steps.
+
+
+!!! tip
+  Unlike the ReplicaSet based weighted canary, a service mesh/ingress based canary leaves the
+  stable as 100% scaled until the rollout completes.
```

Comment:
NOTE: this is actually going to change with PR https://github.com/argoproj/argo-rollouts/pull/1382, which will give the user the option to scale down the stable ReplicaSet, as the canary scales up. 

So this note should get updated when v1.1 is released. Do you still want to add this note?

Reply:
You are right, @jessesuen . Thanks.